### PR TITLE
Allow cache names to be configured

### PR DESCRIPTION
**Pull request by jezhodges**
This is a proposal to add the ability to configure the workbox cache names.

See https://developers.google.com/web/tools/workbox/guides/configure-workbox#configure_cache_names for more information on this.

### CODE REVIEW

File: test/fixture/nuxt.config.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ module.exports = {
     config: {
       debug: true
     },
+    cacheNames: {
```
Comment (pi0) [inline test/fixture/nuxt.config.js]: Can it be an array to allow specifying more than one cache name?

Comment (jezhodges) [reply test/fixture/nuxt.config.js]: I'm not sure if there's a need for it, as this sets the cache names application wide. For example, it allows you to set the default `workbox` cache name to something different. How could this be used with a array setup?

Comment (pi0) [reply test/fixture/nuxt.config.js]: OK, you are right. I checked the docs.

**Comment by jezhodges** [comment]
@pi0 any chance of a release to incorporate these changes please?

**Comment by pi0** [comment]
@jezhodges expect it in a few hours. I've to finish some initial works on workbox-window.

**Pull request merged**